frontend/apps/reader/modules/readermenu.lua

```diff
@@ -254,7 +254,7 @@ function ReaderMenu:setUpdateItemTable()
 
     if Device:supportsScreensaver() then
         local ss_book_settings = {
-            text = _("Exclude this book's content and cover from screensaver"),
+            text = _("Do not show current book cover on lock screen"),
```

Comment:
I think that "Exclude **this** book's ..." makes it clearer it is for **this** book (ie. it has a crappy or offsensive cover).
"Do not show current book" reads as it will apply to any generic "current book".
"Do not show this book..." would be ok I guess.

Reply:
Do not show this book cover on lock screen? I am happy with that.